### PR TITLE
Opanas-added selenium tests

### PR DESCRIPTION
**Pull request by panas224**
Fixed

### CODE REVIEW

File: src/test/java/panas224/Calculation.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package panas224;
```
Comment (toleksyn) [inline src/test/java/panas224/Calculation.java:1]: rename

File: src/test/java/panas224/Calculation.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+public class Calculation {
+    public int calculate(int n)
```
Comment (toleksyn) [inline src/test/java/panas224/Calculation.java:4]: rename

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/test/java/panas224/Calculation.java (hunk 1 above)
Comment (panas224) [reply src/test/java/panas224/Calculation.java]: already renamed

File: src/test/java/opanas/task_01/Calculation.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    {
+        int x0=1;//x^0
+        int r=0;
+        do {
+            r++;
+            x0=(int)Math.pow(2,r);
+
+        }
+        while (x0<=n);
+        return x0;
```
Comment (toleksyn) [inline src/test/java/opanas/task_01/Calculation.java:15]: rename method names and variables

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/test/java/opanas/task_01/CalculationTest.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    @DataProvider(name="Positive digits")
+    public Object[][] GetData(){
+        Object[][] data={
+                {8,4},
+                {4,3},
+        };
+        return data;
+    }
+    @Test(dataProvider = "Positive digits")
+    public void CalculatingTest(int expected,int origin) {
```
Comment (toleksyn) [inline src/test/java/opanas/task_01/CalculationTest.java:19]: test method start with verify()

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/test/java/opanas/task_01/Calculation.java (hunk 3 above)
Comment (panas224) [reply src/test/java/opanas/task_01/Calculation.java]: done

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (toleksyn) [inline pom.xml:1]: remove

File: src/test/java/opanas/task_02/SignInTest.java [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+    public void afterMethod() {
+        driver.manage().deleteAllCookies();
+    }
+    @AfterClass(alwaysRun = true)
+    public void afterClass() {
+        driver.quit();
+    }
+    @Test(description = "In this test we will input the valid data in all fields of Sign in form and look if we will be redirect to our personal account")
+    public void verifyValidData()  {
+        WebElement LogInEmail=driver.findElement(By.id("email"));
```
Comment (toleksyn) [inline src/test/java/opanas/task_02/SignInTest.java:40]: code convention

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 5 above)
Comment (panas224) [reply pom.xml]: done

File: src/test/java/opanas/task_02/SignInTest.java (hunk 6 above)
Comment (panas224) [reply src/test/java/opanas/task_02/SignInTest.java]: done

File: src/test/java/opanas/task_02/SignInTest.java [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+        driver.findElement(By.xpath(".//*[@class='google-sign-in']")).click();
+        driver.manage().timeouts().implicitlyWait(2,TimeUnit.SECONDS);
+        Set<String> handles = driver.getWindowHandles();
+        System.out.println(handles);
+        List<String> siteIdList=new ArrayList<>(handles);
+        driver.switchTo().window(siteIdList.get(1));
+        Assert.assertEquals(driver.findElement(By.xpath("//*[@class='kHn9Lb']")).getText(),"Увійдіть в обліковий запис Google");
+    }
+    @Test()
+    private void verifyForgotPassword()  {
```
Comment (toleksyn) [inline src/test/java/opanas/task_02/SignInTest.java:79]: always public

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/test/java/opanas/task_02/SignInTest.java [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+        driver.manage().timeouts().implicitlyWait(0,TimeUnit.SECONDS);
+        (new WebDriverWait(driver, 10)).until(new ExpectedCondition<Boolean>() {
+            public Boolean apply(WebDriver driver) {
+                return driver.getWindowHandles().size()>1;
+            }
+        }
+        );
+        driver.manage().timeouts().implicitlyWait(10,TimeUnit.SECONDS);
+        Set<String> handles = driver.getWindowHandles();
+        System.out.println(handles);
```
Comment (toleksyn) [inline src/test/java/opanas/task_02/SignInTest.java:80]: no sys out prints

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: build.gradle [hunk 9]
```diff
@@ -10,5 +10,6 @@ repositories {
 }
```
Comment (toleksyn) [inline build.gradle:10]: no need to add custom labels

File: src/test/java/opanas/task_01/CalculationTest.java [hunk 10]
```diff
@@ -0,0 +1,22 @@
+package opanas.task_01;
+
+import org.testng.*;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class CalculationTest {
+    private static Calculation calculation ;
+    @DataProvider(name="Positive digits")
+    public Object[][] GetData(){
```
Comment (toleksyn) [inline src/test/java/opanas/task_01/CalculationTest.java:10]: java code convention

**Review by toleksyn — CHANGES_REQUESTED** [review]

**Pull request merged**